src/xenia/base/clock_win.cc

```diff
@@ -13,15 +13,13 @@
 
 namespace xe {
 
-uint64_t Clock::host_tick_frequency() {
-  static LARGE_INTEGER frequency = {{0}};
-  if (!frequency.QuadPart) {
-    QueryPerformanceFrequency(&frequency);
-  }
+uint64_t Clock::host_tick_frequency_platform() {
+  LARGE_INTEGER frequency = {{0}};
```

Comment:
```suggestion
  LARGE_INTEGER frequency;
```

Pretty sure it doesn't need to be initialized since `QueryPerformanceFrequency` annotates the argument as an output argument.